.ci-support/install.sh

```diff
@@ -1,7 +1,7 @@
 if [ "$(uname)" = "Darwin" ]; then
 PLATFORM=MacOSX
 brew update
-brew upgrade
+# brew upgrade
```

Comment:
Any reason to leave this here? We could also leave a short note explaining why this is commented out

Reply:
I overlooked/forgot it, I say we should eliminate with prejudice.